src/main/java/Engineer/Engineer.java

```diff
@@ -0,0 +1,93 @@
+package engineer;
+import java.util.Scanner;
+import engineer.exception.EngineerException;
+import engineer.task.*;
+import engineer.command.*;
+
+public class Engineer {
+
+    static Scanner scanner = new Scanner(System.in);
+    static TaskManager myTaskManager = new TaskManager();
+    static ChatBotManager myChatBotManager = new ChatBotManager();
+    static TextDivider myTextDivider = new TextDivider();
+
+    public static void printResponse() throws EngineerException{
+        boolean isKeepAsking = true;
+        do {
+            String inputMessage = scanner.nextLine(); // read input
+            String[] words = inputMessage.split(" ");
+            String firstWord = words[0];
+            if (inputMessage.isEmpty()) {
+                throw new EngineerException(myChatBotManager.askForNonEmptyValue());
+            }
```

Comment:
Good job making the [happy path prominent](https://nus-cs2113-ay2425s2.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent).